covid_act_now/tests/conftest.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+from os import listdir, remove
+from os.path import join
+
+from boto3 import Session
+from moto import mock_s3
+import numpy as np
+import pandas as pd
+import pytest
+
+from delphi_utils import read_params
+from delphi_covid_act_now.run import run_module
+
+@pytest.fixture(scope="session")
+def run_as_module():
+    # Clean receiving directory
+    for fname in listdir("receiving"):
+        if fname != ".gitkeep":
+            remove(join("receiving", fname))
+
+    with mock_s3():
+        # Create the fake bucket we will be using
+        params = read_params()
+        aws_credentials = params["aws_credentials"]
+        s3_client = Session(**aws_credentials).client("s3")
+        s3_client.create_bucket(Bucket=params["bucket_name"])
+
+        run_module()
```

Comment:
Is this also mocking the network call to the CAN API? If not, it should -- tests shouldn't hit the network.

Reply:
In the tests, I load a local parquet file `tests/test_data/small_CAN_data.parquet` instead of CAN's actual URL through the `params.json` file.
That file should have the exact same data as in the `CAN_parquet_data` fixture in `conftest.py`
I do feel this is quite duplicated, and maybe I should write `CAN_parquet_data` to a tmp directory and modify the test `params.json` before running the module?

Reply:
ah! i see the tests params template now. all good!